datafusion/physical-plan/src/sorts/sort.rs

```diff
@@ -1005,10 +1005,15 @@ impl DisplayAs for SortExec {
                     None => write!(f, "SortExec: expr=[{}], preserve_partitioning=[{preserve_partitioning}]", self.expr),
                 }
             }
-            DisplayFormatType::TreeRender => {
-                // TODO: collect info
-                write!(f, "")
-            }
+            DisplayFormatType::TreeRender => match self.fetch {
+                Some(fetch) => {
+                    writeln!(f, "fetch={fetch}")?;
```

Comment:
```suggestion
                    writeln!(f, "limit={fetch}")?;
```
I think fetch is more often to be used in implementation, at plan level maybe 'limit' is better 🤔